docs/src/content/docs/resources/community-content.mdx

```diff
@@ -124,5 +124,10 @@ import YouTubeGrid from '~/components/youtube-grid.astro';
 			title: 'Astro Starlight Documentation Template (build custom app docs!)',
 			description: 'Get a new Starlight site up and running in about 5 minutes',
 		},
+		{
+			href: 'https://www.youtube.com/watch?v=12o7WxjAxjM',
+			title: 'Include Astro.js Starlight documentation in a Next.js project with proxies',
```

Comment:
Thanks for the video and the PR @brob — appreciate it!

Going to tweak the `title` just slightly to make it fit onto two lines in most cases:

```suggestion
			title: 'Include Starlight docs in a Next.js project with proxies',
```